app/models/abuse_report.rb

```diff
@@ -124,8 +124,9 @@ def send_report
   end
 
   def attach_work_download(ticket_id)
+    is_not_comments = url[%r{/comments/}, 0].nil?
     work_id = url[%r{/works/(\d+)}, 1]
-    return unless work_id
+    return unless work_id && is_not_comments
```

Comment:
I think this may be easier to read if the variable is simply `comments` and you do the nil check in this line instead of line 127:
```suggestion
    return unless work_id && comments.nil?
```
But this is a subjective style thing so feel free not to do this, I'm already approving the PR.